LibrarySystemBackend/LibrarySystem/LibrarySystem/bin/Debug/net6.0/appsettings.json

```diff
@@ -8,5 +8,11 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "Data Source=CHRISHI\\SQLEXPRESSNEW;Initial Catalog=LibrarySystem;Persist Security Info=True;TrustServerCertificate=True;User ID=sa;Password=ssa"
+  },
+  "JWT": {
+    "ValidAudience": "http://localhost:4200",
+    "ValidIssuer": "http://localhost:5000",
+    "Secret": "JWTAuthenticationHIGHsecuredPasswordVVVp1OH7Xzyr"
```

Comment:
Generate a Random JWT Secret Key. Please refer following article. https://dev.to/tkirwa/generate-a-random-jwt-secret-key-39j4